### DISCUSSION TITLE
Is there any solution on integrating Keycloak and casbin to implement authorization?

### DISCUSSION THREAD

**Discussion by billyhu** [keycloak/keycloak#13047] [Q&A]
As I know, Keycloak can be used for authentication and casbin can be used for authorization.
So is there any solution to integrate them?

**Comment by bs-matil** [comment]
https://casdoor.org/

**Reply by bs-matil** [reply]
AFAIK its aware of OIDC so you can bring this together

**Reply by billyhu** [reply]
Thanks @bs-matil , I will try to investigate casdoor.
